src/app/views/query-response/pivot-items/pivot-items.tsx

```diff
@@ -19,7 +19,7 @@ import { queryResponseStyles } from '../queryResponse.styles';
 import { Response } from '../response';
 import { Snippets } from '../snippets';
 
-export const getPivotItems = () => {
+export const GetPivotItems = () => {
```

Comment:
Why the change of function name from camel case?

Reply:
The function is returning components and the recommended convention is to have all of these functions camel-cased. SonarCloud also flagged it as a bug
<img width="1179" alt="image" src="https://user-images.githubusercontent.com/45680252/182313783-3155b9d5-a4e8-4353-b75e-01f60a8dcd0e.png">

https://sonarcloud.io/project/issues?resolutions=FIXED&pullRequest=1958&id=microsoftgraph_microsoft-graph-explorer-v4&open=AYJYe_3T5cxs88NX26G2

Reply:
You mean pascal-cased?